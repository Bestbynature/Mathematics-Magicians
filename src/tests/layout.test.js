import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import React from 'react';
import Layout from '../components/layout';

describe('Layout', () => {
  it('renders correctly', () => {
    const { container } = render(/* #__PURE__ */React.createElement(Layout, null));
    expect(container).toMatchSnapshot();
  });

  it('renders the correct text', () => {
    const {
      getByText,
    } = render(/* #__PURE__ */React.createElement(Layout, null));
    expect(getByText('Layout')).toBeInTheDocument();
  });
});
