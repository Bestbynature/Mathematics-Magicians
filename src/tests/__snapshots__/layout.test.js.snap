// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div>
  <h1>
    Layout
  </h1>
</div>
`;
